app/webutil.js

```diff
@@ -162,9 +163,9 @@ export function readSetting (name, defaultValue) {
 
 export function eraseSetting (name) {
     "use strict";
+    settings[name] = undefined;
```

Comment:
Why not `delete` to get things fully back to before it was ever set?

Reply:
If the setting was deleted, then the next time it is read it would be restored back to the value in localstorage.  If localstorage was modified by another tab between the erase and the next read, then the read value would be inconsistent.

It may be a moot point though since it seems that `eraseSetting` is never used.  Would it just be better to remove it instead?

Reply:
Not entirely clear what we want here since it isn't called. But if I read the code correctly this will nuke it from local storage as well. So the possibly confusing case is the one you describe, when another tab does a write after the first tab has done an erase.

I think a `delete` would be safer for now and we can look over this once the API starts getting use. You could add a comment about this confusion there if you'd like.